client/views/windows/help.tpl

```diff
@@ -47,6 +47,15 @@
 
 	<h2>Keyboard Shortcuts</h2>
 
+	<div class="help-item">
+		<div class="subject">
+			<kbd class="key-all">Alt</kbd><kbd class="key-apple">⌥</kbd> + <kbd>Shift</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>
```

Comment:
On the macOS version, it currently looks like this:

<img width="163" alt="screen shot 2018-03-13 at 19 22 35" src="https://user-images.githubusercontent.com/113730/37378732-d4b45f32-2706-11e8-9aa0-89ec0bab5533.png">

We tend to use the ⌥/⌘ icons to match how macOS natively shows these things:

<img width="200" alt="screen shot 2018-03-13 at 19 24 09" src="https://user-images.githubusercontent.com/113730/37375019-276bb40e-26f4-11e8-9295-e3764b93316d.png">

For Shift, that would be ⇧ (`&#x21E7; – &#8679;`). Would you mind switching to that one?

Reply:
So I tried that icon at first but the rendering looked really bad at the small size it appeared… since you’re reformatting things with #2221 maybe you can try it out and see if it works better with the bigger font size.

Reply:
Yeah, that's part of what drove me to do #2221. Go with that one, and I'll reformat in that other PR once yours is merged :)

Reply:
Actually, feel free to leave as is, I'll switch in my PR, don't worry about this in your PR, it's so close to completion already.